app/controllers/api/v2/providers_controller.rb

```diff
@@ -6,7 +6,7 @@ class ProvidersController < ApplicationController
       def index
         authorize Provider
         providers = policy_scope(Provider)
-        providers = providers.where(id: @user.providers) if @user.present?
+        providers = providers.where(id: @user.providers).order(:provider_name) if @user.present?
```

Comment:
Placing the `order` on this line means that if `@user` isn't specified, the results are unordered...